src/components/Icon/assets/Forum.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="17" viewBox="0 0 18 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Make sure the SVG viewport is a square, and the icon is centered horizontally and vertically in it.
i.e. `viewBox="0 0 18 17"` -> `viewBox="0 -0.5 18 18"`